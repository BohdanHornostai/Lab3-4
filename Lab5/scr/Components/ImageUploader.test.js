import React from 'react'
import {render, cleanup} from '@testing-library/react'
import ImageUploader from './ImageUploader'

 afterEach(cleanup)
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<ImageUploader />)

    expect(asFragment(<ImageUploader />)).toMatchSnapshot()
   })