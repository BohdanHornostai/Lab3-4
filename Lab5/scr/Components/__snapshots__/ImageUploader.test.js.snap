// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="previewComponent cA cAlabel"
    >
      <form>
        <input
          accept="image/*"
          class="inputfile"
          id="file"
          name="file"
          type="file"
        />
        <label
          for="file"
        >
          Choose a file
        </label>
      </form>
      <button
        class="submitButton"
        type="submit"
      >
        Upload Image
      </button>
    </div>
  </div>
</DocumentFragment>
`;
